### ISSUE TITLE
Pagination?

### ISSUE THREAD

**Issue by nielsk** [aezell/vapor1994#3]
Is there any way to get pagination in the theme?

**Comment by w4rner** [comment]
good question, my page hasn't responded to setting 5 for the new https://github.com/microdotblog/plugin-paginate

**Comment by aezell** [comment]
I can look into this. I definitely did not take any pagination options into account. It might be about providing a place for the paginate plugin to hook into.

**Comment by w4rner** [comment]
Yeah it's probably like Manton's edit to allow meta tags in: a standard line to allow M.b plugins to do their magic!

------- Original Message -------

**Comment by kimberlyhirsh** [comment]
I'd like to bump this feature request. I really want pagination and love this theme, but might switch away from it until pagination works.

**Comment by aezell** [comment]
I can work on this over the weekend. I completely forgot about this request. Thanks for bumping it!
